gtsfm/utils/metrics.py

```diff
@@ -176,6 +176,8 @@ def compute_averaging_metrics(
     gt_wRi_list, gt_wti_list = get_rotations_translations_from_poses(gt_wTi_list)
 
     metrics = []
+    metrics.append(GtsfmMetric(name="num_rotations_computed", data=len(wRi_list)))
```

Comment:
Thanks for the PR Ayush.

should it instead be the number of not None entries? Can you double check it works as expected on some data?

Reply:
Thanks John. Fixed it now